Website/src/rendering/src/components/SessionInformation.tsx

```diff
@@ -60,6 +62,38 @@ const SessionInformation = (props: SessionInformationProps): JSX.Element => {
     ? 'Speaker'
     : 'Speakers';
 
+  const room =
```

Comment:
This is what I had in mind to improve the JSX readability.
When I recently worked on the OrderSummary, I had issues where the HTML element next to where these consts are used were getting their `className` not rendered to the DOM. It took me a while to fin a workaround. This seems to be fragile. Keep an eye on that issue when working on this story.